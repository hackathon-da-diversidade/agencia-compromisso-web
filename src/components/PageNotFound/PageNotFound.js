import React, { Component } from 'react';
import Header from '../Header/Header';

class PageNotFound extends Component {
  render() {
    return (
      <>
        <Header />
        <h1>Ops!</h1>
        <span>Página não encontrada.</span>
      </>
    );
  }
}

export default PageNotFound;
