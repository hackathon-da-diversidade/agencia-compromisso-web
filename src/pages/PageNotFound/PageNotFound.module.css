h1 {
  font-size: 22px;
  font-weight: bold;
  margin-top: 2em;
}

span {
  font-size: 16px;
}
