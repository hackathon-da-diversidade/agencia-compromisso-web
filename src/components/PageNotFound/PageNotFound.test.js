import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import PageNotFound from './PageNotFound';

it('loads component', () => {
  const { getByText } = render(
    <BrowserRouter>
      <PageNotFound />
    </BrowserRouter>
  );

  expect(getByText('Página não encontrada.')).toBeInTheDocument();
});
