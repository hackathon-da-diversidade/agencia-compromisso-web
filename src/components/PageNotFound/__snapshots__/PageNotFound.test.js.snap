// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="Header"
  >
    <a
      class="HeaderBack"
      href="/"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
        />
      </svg>
    </a>
    <img
      alt="ícone estilizado com o perfil de uma mulher com uma fita métrica no black power"
      class="HeaderIcon"
      src="icon.png"
    />
    <img
      alt="banner da Agência Compromisso"
      class="HeaderBanner"
      src="banner.png"
    />
    <h2
      class="HeaderTitle"
    />
  </div>
  <h1>
    Ops!
  </h1>
  <span>
    Página não encontrada.
  </span>
</DocumentFragment>
`;
